components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -193,4 +247,109 @@ private RecoveryResponse recover(RecoveryRequest recoveryRequest, String tenantD
+            localVarHeaderParams.putAll(headers);
+        }
+        Map<String, Object> localVarFormParams = new HashMap<String, Object>();
+        final String[] localVarAccepts = {
+                "application/json"
+        };
+        final String localVarAccept = apiClient.selectHeaderAccept(localVarAccepts);
+        final String[] localVarContentTypes = {
+                "application/json"
+        };
+        final String localVarContentType = apiClient.selectHeaderContentType(localVarContentTypes);
+        String[] localVarAuthNames = new String[]{};
+        GenericType<ResendResponse> localVarReturnType = new GenericType<ResendResponse>() {
+        };
+        return apiClient.invokeAPI(localVarPath, "POST", localVarQueryParams, resendRequest,
+                localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames,
+                localVarReturnType);
+    }
+
+    private ConfirmResponse confirm(ConfirmRequest confirmRequest, String tenantDomain, Map<String, String> headers,
+                                    String localVarPath) throws ApiException {
+
+        // Verify if the required parameter 'recoveryRequest' is set.
+        if (confirmRequest == null) {
+            throw new ApiException(400, "Missing the required parameter 'confirmRequest' when requesting recovery");
+        }
+        if (StringUtils.isBlank(tenantDomain)) {
+            tenantDomain = MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
+        }
+        basePath = IdentityManagementEndpointUtil.getBasePath(tenantDomain,
+                IdentityManagementEndpointConstants.UserInfoRecovery.RECOVERY_API_V2_RELATIVE_PATH);
+        apiClient.setBasePath(basePath);
+        List<Pair> localVarQueryParams = new ArrayList<Pair>();
+        Map<String, String> localVarHeaderParams = new HashMap<String, String>();
+        if (MapUtils.isNotEmpty(headers)) {
+            localVarHeaderParams.putAll(headers);
+        }
+        Map<String, Object> localVarFormParams = new HashMap<String, Object>();
+        final String[] localVarAccepts = {
+                "application/json"
+        };
+        final String localVarAccept = apiClient.selectHeaderAccept(localVarAccepts);
+        final String[] localVarContentTypes = {
+                "application/json"
+        };
+        final String localVarContentType = apiClient.selectHeaderContentType(localVarContentTypes);
+        String[] localVarAuthNames = new String[]{};
+        GenericType<ConfirmResponse> localVarReturnType = new GenericType<ConfirmResponse>() {
+        };
+        return apiClient.invokeAPI(localVarPath, "POST", localVarQueryParams, confirmRequest,
+                localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames,
+                localVarReturnType);
+    }
+
+    private ResetResponse reset(ResetRequest resetRequest, String tenantDomain, Map<String, String> headers,
```

Comment:
refer comments for `resend` method

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/2c29350aad74d55ec4a6243007ac968e35cc02f1